plugins/popeye.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: popeye
+spec:
+  homepage: https://popeyecli.io
+  shortDescription: 👀 A Kubernetes cluster resource sanitizer
```

Comment:
Let's avoid emojis as they aren't available everywhere. (e.g. windows terminals).

Also you repeat `Kubernetes` word which is obvious in this context. 

I'll help you rewrite this:

> Scan cluster and resources for potential issues

Reply:
> Let's avoid emojis as they aren't available everywhere.

Just a question: would we accept ascii art?

Reply:
Not in shortDescription since it’s meant to be one-line. 

For description, also probably not as we might show it on the website someday.